src/Tribe/Views/V2/Views/Traits/HTML_Cache.php

```diff
@@ -196,4 +200,152 @@ protected function should_enable_html_cache( $context ) {
 		// In all other cases, let's cache it!
 		return true;
 	}
+
+	/**
+	 * Get the list of fields/input we will do replacement for HTML Cache.
+	 *
+	 * @since 5.0.0
+	 *
+	 * @return array   List of fields/input that we are going to replace.
+	 */
+	protected function get_view_nonce_fields() {
+		$nonces = [
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 845b35765445a6875431c15d35a078de</sub>